wordpress/5/versions

```diff
@@ -1,3 +1,4 @@
 5.0.1|none
 5.1.1|none
 5.2.4|none
+5.3|none
```

Comment:
@laszlof we need to test to ensure this works.  We *may* need the "full" version 5.3.0 but testing will tell.